tests/e2e-pw/specs/merchant/merchant-admin-account-balance.spec.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { useMerchant } from '../../utils/helpers';
+
+test.describe(
+	'Merchant account balance overview for single deposit currency accounts',
+	() => {
+		// Use the merchant user for this test suite.
+		useMerchant();
+
+		test(
+			'View the total and available account balance for a single deposit currency (USD)',
+			{
+				tag: '@critical',
+			},
+			async ( { page } ) => {
+				await test.step(
+					'Navigate to the Payments Overview screen',
+					async () => {
+						await page.goto(
+							'/wp-admin/admin.php?page=wc-admin&path=/payments/overview'
+						);
+					}
+				);
+
+				await test.step(
+					'Observe the total account balance, ensuring it has a dollar value with the correct currency formatting',
+					async () => {
+						const totalBalanceValue = page.getByLabel(
+							'Total balance',
+							{
+								exact: true,
+							}
+						);
+
+						// Match the total balance value to the USD format $1*
+						// Intentionally not expecting a specific value to allow for different values in different environments.
+						await expect( totalBalanceValue ).toHaveText( /\$\d+/ );
```

Comment:
Hi @Jinksi!

Finally fixed my E2E env. I ran this test locally and it failed because of the currency, it's expecting USD but my site has EUR:
```
@
merchant-admin-account-balance.spec.ts:44
Error: Timed out 10000ms waiting for expect(locator).toHaveText(expected)

Locator: getByLabel('Total balance', { exact: true })
Expected pattern: /\$\d+/
Received string:  "€179.35"
```

I don't really know why this happened (maybe this is set automatically depending on the location of your IP?), should we force the USD currency in the site setup?

Reply:
That is interesting. I'm based in AU but my e2e store is set up with USD.

I've modified the regex and now it _should_ match any currency format in 9b52595.

Pulling this thread: we're going to need more control over merchant details like this if we want to test different flows that apply to specific store configurations – e.g. single deposit currency, multiple deposit currencies, instant deposit eligibility.

Something like DB resetting for a test suite pcShBQ-1OP-p2. Or, for WooPayments it might be more suitable to switch between multiple pre-configured merchant accounts.

Reply:
> I've modified the regex and now it should match any currency format in https://github.com/Automattic/woocommerce-payments/commit/9b5259501e6f359a7b20a5fdc69e057dff020ddc.

Thanks!

Btw, I think we can follow a more robust approach by getting the currency from WooCommerce options table (we can run the WP CLI command `wp option get woocommerce_currency --allow-root`) in a `beforeAll` hook and then use it in the regex pattern in the test.

> Pulling this thread: we're going to need more control over merchant details like this if we want to test different flows that apply to specific store configurations – e.g. single deposit currency, multiple deposit currencies, instant deposit eligibility. This would need to be resolved as part of issue https://github.com/Automattic/woocommerce-payments/issues/9188.

Totally agree! Tests should be independent, deterministic and repeatable so the more control we have on the environment and preconditions, the better 👍  Thanks for creating the issue!

As a note, I ran `wp option get woocommerce_currency --allow-root` in my `wcp_e2e_wordpress` container and it returned USD, but the test failed because it shoed the amount with EUR. My local environment could be playing with me again or I must be doing something wrong here 😆

Reply:
_TIL_ the WooPayments account used for e2e testing **locally** is defined by the `E2E_WCPAY_STRIPE_ACCOUNT_ID` in `tests/e2e/config/local.env`

So we change this to an account with different capabilities, e.g. `acct_1MoxnYC15a3WbJ1t` for a multi-deposit-currency test account with `nok` currency as default.

Reply:
I had a crack at detecting the specific account currency on a separate branch `e2e/add-merchant-account-balance-overview-pw-tests-specific-currency`, commit https://github.com/Automattic/woocommerce-payments/commit/b26f05cc6671c3e876aa76193f29523c90ae2a18.

After trying this, checking the correct currency formatting may not be worth the added complexity → increases the potential flakiness of the test. Ensuring that a valid `Total balance` and `Available balance` value is present may be sufficient for this test. We can rely on our integration/unit testing to check for valid currency formatting.